oh_staff_ui/templates/oh_staff_ui/upload_file.html

```diff
@@ -44,6 +44,7 @@
     <th>Added by</th>
     <th>Original name</th>
     <th>New name & location</th>
+    <th>Size</th>
```

Comment:
Either here or in the table itself, can we say that the the size is given in bytes?

Reply:
Good catch.  I've just pushed a change which adds "bytes" to the display.   We might move it to the table header later, when this gets restyled.